### PR TITLE
Three new cwl files, and other changes

### PR DESCRIPTION
**Pull request by muzimuzhi**
Main changes:
 - Added three new cwl files for sub-package of tikz-pgf: `pgfrcs`, `pgfkeys`, and `pgfmath`.
 - Removed corresponding lines in `pgfcore.cwl`.
 - Removed duplicate lines from multiple cwl files, most of which are autogenerated cwl lines for tex primitives.

Minor changes:
 - Updated some pgf commands in `pgfcore.cwl`.
 - Updated some other cwl files.
 - Resorted cwl file list in `completion.qrc`. Only apparent unsorted lines were adjusted.

_Edit:_ All commands in those newly added cwl files are marked unusual (end with `#*`), except
 - `\pgfkeys{key list}` and `\pgfqkeys{default path}{key list}` in `pgfkeys.cwl`
 - `\pgfmathprint{real number}`, `\pgfmathparse{math expression}`, and `\pgfmathresult` in `pgfmath.cwl`

### CODE REVIEW

File: completion/hyperref.cwl [hunk 1]
```diff
@@ -129,8 +129,8 @@ xetex
 \hyperref[label%ref]{URL}{category}{name}{text}#M
 \hyperref[label%ref]{text}
 \hyperref{text}
-\hyperlink{name}{text}
-\hypertarget{name}{text}
+\hyperlink{target name}{link text}
+\hypertarget{target name}{anchor text}
```
Comment (sunderme) [inline completion/hyperref.cwl:133]: i would assume that spell checking is now disabled for link/anchor text ... %text?

Comment (muzimuzhi) [reply completion/hyperref.cwl]: Ah yes. Updated. I often forget those reserved argument names.

**Comment by muzimuzhi** [comment]
Added list of commands in new cwl files not end with `#*` to original comment.

**Comment by sunderme** [comment]
thanks

**Pull request merged**